tensorflow_serving/model_servers/tensorflow_model_server_test.py

```diff
@@ -163,7 +205,41 @@ def _TestBadModelUpconvertedSavedModel(self):
   def _TestBadModelSessionBundle(self):
     """Test Predict against a bad SessionBundle model export."""
     self._TestBadModel(use_saved_model=False)
-
-
+    
+  def testGoodFileConfig(self):
+    """Test server model configuration from file works with valid configuration"""
+    atexit.register(self.TerminateProcs)
+    model_server_address = self.RunServerWithModelConfigFile(PickUnusedPort(),
+                                                             self._GetGoodConfigFile(),
+                                                             False) # use_saved_model
```

Comment:
Can you set use_saved_model=True. We are preferring SavedModel in new code and will soon deprecate SessionBundle.